BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -32,6 +32,7 @@ public struct ScrollView: Element {
 
     public var delaysContentTouches: Bool = true
 
+    public var setContentOffsetTrigger: ScrollTrigger?
```

Comment:
Could probably just call this `contentOffsetTrigger` to avoid the awkwardness of a var that starts with "set".